yt/frontends/arepo/tests/test_outputs.py

```diff
@@ -55,12 +58,21 @@ def test_tng_hdf5():
     ]
 )
 
-
 @requires_ds(tng59_h5)
 def test_arepo_tng59():
-    ds = data_dir_load(tng59_h5)
+    ds = data_dir_load(tng59_h5, kwargs = {'bounding_box': _tng59_bbox})
     for test in sph_answer(ds, 'halo_59', 10107142,
                            tng59_fields):
```

Comment:
How come adding the bounding box didn't change the number of particles, `10107142`?

Reply:
Great question.  I set up the bounding box to include all of the particles.  The TNG Halo we have has a *domain* set to be the full domain of the TNG simulation, but it's just one halo.  So things like `"c"` were going to the center of the domain, the `unitary` unit was a bit strange, etc.  Now using this, it's constrained to just the location of the halo.